test/OpenTelemetry.Exporter.Prometheus.Tests/PrometheusExporterMiddlewareTests.cs

```diff
@@ -26,13 +26,14 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using OpenTelemetry.Metrics;
+using OpenTelemetry.Tests;
 using Xunit;
 
 namespace OpenTelemetry.Exporter.Prometheus.Tests
 {
     public sealed class PrometheusExporterMiddlewareTests
     {
-        private const string MeterName = "PrometheusExporterMiddlewareTests.Meter";
+        private static readonly string MeterName = Utils.GetCurrentMethodName();
```

Comment:
Curious - what would be the actual value? 😄

Reply:
So am I 😁
I will update the description with a list of all the meter names once I have printed them out to ensure uniqueness

Reply:
The answer is `OpenTelemetry.Exporter.Prometheus.Tests.PrometheusExporterMiddlewareTests..cctor`

Reply:
That's good to know, thanks @mic-max!